ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -196,7 +196,7 @@ defaults <- node_group <- node_group.node <- node
     # Uplink switches (list). | Required.
     uplink_switches: [ < uplink_switch_inventory_hostname 01 >, < uplink_switch_inventory_hostname 02 > ]
 
-    # Number of interfaces towards uplink switches | Optional
+    # Number of uplink switches | Optional
```

Comment:
Perhaps, on setting this value to allow for growth?
something like, but I'm not satisfied with what I came up with!

```yaml
# Adjust to the potential fabric growth to allow expansion without re-assigning IP addresses on p2p uplinks.
```

Reply:
Updated according to Claus suggestion.